src/lib/logical_query_plan/lqp_translator.cpp

```diff
@@ -92,7 +92,7 @@ std::shared_ptr<AbstractOperator> LQPTranslator::translate_node(const std::share
    * would result in multiple operators created from predicate_c and thus in performance drops.
    *
    * Deduplication:
-   * _operator_by_lqp_node compares entries by value (i.e., AbstractOperator::operator==), not by identity
+   * _operator_by_lqp_node_cache compares entries by value (i.e., AbstractOperator::operator==), not by identity
```

Comment:
For me, a "cache" is something that simply stores the result of an idempotent computation. Clearing the cache at any point should have no effect on the result of the algorithm (here: the LQP translation) other than its execution duration. As such, in my understanding, this is not a cache.

Reply:
> Clearing the cache at any point should have no effect on the result of the algorithm (here: the LQP translation) other than its execution duration.

Good point. I revert this change.